dag/walkthrough.yml

```diff
@@ -9,3 +9,10 @@ steps:
   data://explorers/dummy/2020-01-01/dummy:
   - data://garden/dummy/2020-01-01/dummy
 
+
+  data://meadow/artificial_intelligence/2023-06-14/papers_with_code_math:
+  - snapshot://artificial_intelligence/2023-06-14/papers_with_code_math.html
+  data://garden/artificial_intelligence/2023-06-14/papers_with_code_math:
+  - data://meadow/artificial_intelligence/2023-06-14/papers_with_code_math
+  data://grapher/artificial_intelligence/2023-06-14/papers_with_code_math:
+  - data://garden/artificial_intelligence/2023-06-14/papers_with_code_math
```

Comment:
I'd move this to the main dag. Or maybe even to a dedicated `technology` or `ai` new dag.